.github/workflows/ci.yml

```diff
@@ -7,30 +7,29 @@ jobs:
       runs-on: ubuntu-latest
       steps:
          -
-            uses: actions/checkout@v3
+            uses: actions/checkout@v4
             with:
                fetch-depth: 0 # Fetch all history
          -
-            uses: actions/setup-node@v3
+            uses: actions/setup-node@v4
             with:
                node-version-file: '.nvmrc'
-         - run: npm i -g npm@8.5.5
          - run: npm ci
+         - run: npm run check-node-version
```

Comment:
I think this might fail in the Node 16 pipeline, since we're taking out the installation step. Let's discuss after team meeting?